src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -350,6 +350,11 @@ export default class MutationMapper<
                             counts={this.mutationCountsByProteinImpactType}
                             onSelect={this.onProteinImpactTypeSelect}
                             onClickSettingMenu={this.props.onClickSettingMenu}
+                            annotatedProteinImpactTypeFilter={this.store.dataStore.dataFilters.find(
+                                filter =>
+                                    filter.type ===
+                                    ANNOTATED_PROTEIN_IMPACT_FILTER_TYPE
+                            )}
```

Comment:
Might be better to add a `@computed get annotatedProteinImpactTypeFilter` for this.